grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -161,17 +161,23 @@ def setUp(self):
     super(FakeOsqueryFlowTest, self).setUp()
     self.client_id = self.SetupClient(0)
 
-  def _InitializeFlow(self, query: str, file_collection_columns: List[str]):
+  def _InitializeFlow(
+      self,
+      query: str,
+      file_collection_columns: List[str],
+      **kwargs,
```

Comment:
Since you add just one parameter, I would name and type it explicitly rather than use `**kwargs`. At this point it is questionable whether we don't just prefer to use `**kwargs` for everything, but this is something for another PR (if any).

Reply:
My intention was to use explicit argument names and types for flow arguments (query and columns to collect files from), and `**kwargs` for other "internal" parameters (`check_flow_errors` in this case, which is used by the test library). With this in mind, would you still prefer to have `check_flow_errors` as an explicit argument (or all passed as `kwargs`)?

Reply:
I forgot that `check_flow_errors` is part of the flow runner arguments, not a flow argument itself. In that case, I think it is fine to leave it as it is.